lib/components/app/batch-routing-panel.js

```diff
@@ -28,14 +28,7 @@ class BatchRoutingPanel extends Component {
         })
 
     return (
-      <ViewerContainer
-        className="batch-routing-panel"
-        style={{
-          display: 'flex',
```

Comment:
The removal of this causes the white block to appear at the bottom of an incomplete load as you can see here.

https://user-images.githubusercontent.com/86619099/217650871-6f6e8905-b1fc-4893-bf66-fbf280a196ad.mov

Reply:
Okay. It's back in https://github.com/opentripplanner/otp-react-redux/pull/783/commits/6832c0465b2b33e95266419e2ab2d7c6ed17f413.